test/functional/from_agent/test_create_tls_container.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+'''
+test_requirements = {'devices':         [],
+                     'openstack_infra': []}
+
+'''
+# Copyright 2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from barbicanclient import client
+from keystoneauth1 import identity
+from keystoneauth1 import session
+from pytest import symbols as symbol_data
+
+from f5_openstack_agent.lbaasv2.drivers.bigip import barbican_cert
```

Comment:
I can accept this test, because we have designs to move the barbican_cert manager into the driver very soon.